strings/base_includes.h

```diff
@@ -27,8 +27,10 @@
 
 #if __has_include(<windowsnumerics.impl.h>)
 #define WINRT_IMPL_NUMERICS
+#if __has_include(<directxmath.h>)
```

Comment:
Is it intentional that directxmath.h is only included if windowsnumeric.impl.h is available?

Reply:
This was done to work around a C++ modules bug. It's required by numerics but not by cppwinrt itself.

Reply:
Feel free to remove it if it helps.

Reply:
Ah, can probably keep for now and I'll keep it in mind when I look into modules again soon